packages/cli/src/databases/entities/WorkflowEntity.ts

```diff
@@ -100,6 +100,7 @@ export class WorkflowEntity extends AbstractEntity implements IWorkflowDb {
 	@AfterInsert()
 	setHash(): void {
 		const { name, active, nodes, connections, settings, staticData, pinData } = this;
+		if (!connections) return;
```

Comment:
Why do we skip the hashing if there are no connections?

Reply:
`connections` is `null` in he objects created in `/rest/workflows` because we don't query for that column. This is a hacky way to disable hash calculation for all the workflows objects for the listing call. 
Anywhere else where we fetch all the columns, `connections` never falsey, and the hash calculation works as usual.

Reply:
Gotcha, so I think we should be leaving a comment here just so we don't forget about it int he future wdyt?